packages/KDateTimePicker/KDateTimePicker.vue

```diff
@@ -457,19 +471,15 @@ export default defineComponent({
 </script>
 
 <style lang="scss">
-$margin: .2rem;
-$timepicker-min-width: 24rem;
+$timepicker-min-width: 22rem;
+$margin: 0.2rem;
```

Comment:
I didn't catch this in the original PR, but you need to remove **ALL** `rem` units from the component and replace with pixels or a relative unit (e.g. `%`)

Since we cannot control the default font size of the root document element in a consuming app, we cannot assume `rem` units will display as desired. 

Please change all `rem` units throughout the component and styles that were added.